src/app/components/accessibility/accessibility.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'exui-accessibility',
+    templateUrl: './accessibility.component.html'
+})
+export class AccessibilityComponent {
+
+  date = '19 September 2019';
```

Comment:
what's this hardcoded date string for?

Reply:
it is the date when the testing done, which is part of the html content. I make it as a variable so we could change that easily